app/controllers/application_controller.rb

```diff
@@ -1,3 +1,13 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  def receive
+    recieve = 'hogehogehogehoge'
+
+  end
+
+
+  def defualt
+    defualt = 'default programmme'
```

Comment:
__[Misspell]__

"defualt" is a misspelling of "default"


<!-- Comment by [Sider](https://sider.review) -->